tools/api-markdown-documenter/src/utilities/ApiItemUtilities.ts

```diff
@@ -241,22 +241,15 @@ export function filterByKind(apiItems: readonly ApiItem[], kinds: ApiItemKind[])
 /**
  * Gets the release tag associated with the provided API item, if the item's documentation contained one.
  *
+ * @remarks
+ * Note: getting the exact tag of an item is generally not useful.
+ * Use `getEffectiveReleaseTag` instead to get the effective release level, accounting for inheritance.
+ *
  * @param apiItem - The API item whose release tag is being queried.
  *
  * @returns The associated release tag, if it exists. Will return `None` if no tag is present.
- *
- * @privateRemarks
- * TODO: No one should really use this. They should use `getEffectiveReleaseTag` instead.
- * This includes the docs we generate - we shouldn't label an interface member as `@public` if the interface itself is
- * `@beta`, for example, even if that member is directly tagged `@public`.
- *
- * @deprecated
- * Generally, getting the exact tag of an item is not useful. Use `getEffectiveReleaseTag` instead to get the
- * effective release level, accounting for inheritance.
- *
- * @public
  */
-export function getReleaseTag(apiItem: ApiItem): ReleaseTag {
+function getReleaseTag(apiItem: ApiItem): ReleaseTag {
```

Comment:
nit: It might be useful to leave the deprecated tag here despite it not being exported. Also is it better to remove this entirely if it's no longer in use / does not work as expected?

Reply:
I imagine it's still used internally by the package somewhere? Otherwise I agree with Sean, removal seems fine.

Reply:
It's used by `getEffectiveReleaseLevel` as an implementation detail. This functionality still has meaning IMO, it just isn't generally useful by consumers of this library.